deno-runtime/handlers/app/construct.ts

```diff
@@ -31,11 +31,12 @@ function wrapAppCode(code: string): (require: (module: string) => unknown) => Pr
     return new Function(
         'require',
         `
+        const Buffer = require('node:buffer');
         const exports = {};
         const module = { exports };
-        const result = (async (exports,module,require,globalThis,Deno) => {
+        const result = (async (exports,module,Buffer,require,globalThis) => {
```

Comment:
```suggestion
        const result = (async (exports,module,require,Buffer,globalThis, Deno) => {
```

The order of parameters in the call is different, and we should still "shadow" the `Deno` name so it's not so easily accessible